.github/workflows/switch.yml

```diff
@@ -16,46 +16,22 @@ jobs:
     container:
       image: 'devkitpro/devkita64'
     steps:
-      - name: Install git
-        run: |
-          echo "deb http://ftp.debian.org/debian stretch-backports main" | sudo tee /etc/apt/sources.list.d/stretch-backports.list
-          sudo apt update
-          sudo apt install -t stretch-backports git -y
       - name: Checkout code
         uses: actions/checkout@v2
         with:
           # We need 0 so we get all commits for mtime!
           fetch-depth: 0
           path: "./stk-code"
-      - name: Checkout OpenAL
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-          path: "./stk-code/lib/openal"
-          repository: "kcat/openal-soft"
-      - name: Checkout Harfbuzz
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-          path: "./stk-code/lib/harfbuzz"
-          repository: "harfbuzz/harfbuzz"
       - name: Grab assets
         run: |
-          sudo apt update
-          sudo apt install unzip zip -y
           wget -q https://github.com/supertuxkart/stk-assets-mobile/releases/download/git/stk-assets-full.zip
           unzip -q stk-assets-full.zip -d stk-assets
           rm stk-assets-full.zip
       - name: Restore modified date
         run: |
-          sudo apt install -y git-restore-mtime
           cd stk-code
           git restore-mtime .
-          cd lib/openal
-          git restore-mtime .
-          cd ../../lib/harfbuzz
-          git restore-mtime .
```

Comment:
Why was this removed? This breaks caching (make uses last modified dates) leading to really long compile times

Reply:
Other than this I don't see any big issues

Reply:
This is just removing the libs that were being compiled but are now packages in the devkita64 docker. restore mtime is still there for stk-code - https://github.com/WinterMute/stk-code/blob/fix-switch-hacks/.github/workflows/switch.yml#L33

Reply:
Yeah sorry, I realised that, I thought I left a comment saying that when I approved the review